GT4PY_VERSION.txt

```diff
@@ -1 +1 @@
-v26
+3053f82e5ab1acc6108b739df9cc6cb178ae78b3
```

Comment:
we both know this, but just want to add the comment for anyone else's benefit, this is meant to be temporary until a new gt4py tag is generated, at which point we will update fv3core master to that